src/devtools/client/debugger/src/components/PrimaryPanes/Sources.css

```diff
@@ -11,7 +11,7 @@
 }
 
 .sources-list {
-  height: 100%;
+  /* height: 100%; */
```

Comment:
this does not seem dark theme related. Probably should be it's own PR

Reply:
It is because this line is forcing a visual flaw in dark theme and I couldn't see any regression in light when I removed it

Reply:
Cool - i think @jaril wants to revisit this as well so that the source pane by default is a bit larger

Reply:
Ah good to know, thanks. I suspected as much, which is why I commented it out. CC'ing @jaril to make sure I'm not being dumb.